### ISSUE TITLE
Trained 1080p model?

### ISSUE THREAD

**Issue by kmonachopoulos** [Byronnar/tensorflow-serving-yolov3#9]
Is it possible to share the trained 1080p model (in checkpoint or protobuf) ??

**Comment by Byronnar** [comment]
I am sorry, for some reasons, the model is temporarily inconvenient to expose, you can train your own model, parameters may need to be adjusted.

**Comment by kmonachopoulos** [comment]
@Byronnar I am trying to export a 1080p using pretrained darknet weights and the Python graph definition but changing the input resolution brakes the export process. 

I understand that if I change the input resolution I have to somehow change the network definition (architecture). Do you know which parts should I change ? I don’t mind if the 1080p network that you have does not produce good scores, I just want to check the network structure to see the difference from current version.

Thanks again 🤟

**Comment by Byronnar** [comment]
your input is 1080*1080?

**Comment by kmonachopoulos** [comment]
The target resolution is 1920 x 1080

**Comment by Byronnar** [comment]
hello, You can modify the yolov3.py file at lines 47
        with tf.variable_scope('pred_multi_scale'):
            self.pred_multi_scale = tf.concat([tf.reshape(self.pred_sbbox, [-1, 19, 19, 85]),
                                               tf.reshape(self.pred_mbbox, [-1, 19, 19, 85]),
                                               tf.reshape(self.pred_lbbox, [-1, 19, 19, 85])], axis=0, name='concat')
what is your classes?
The darknet pretrained file should be first changed to ckpt model.

**Comment by kmonachopoulos** [comment]
Thank you for the information. However, when I try to inference using the extracted network through image_demo_Chinese.py the script breaks... 
I intend to use the same classes with the ones that used on the pre-trained network (so coco->80 classes).

**Comment by Byronnar** [comment]
Have you used the frezz.py script？ And then use the image_demo .py script.
